generators/app/templates/index.js

```diff
@@ -0,0 +1,14 @@
+const config = {
```

Comment:
This could replace existent index.js file of project, maybe we can ignore this template for now and tell user that define some variables in config file